package.json

```diff
@@ -1,9 +1,15 @@
 {
   "private": true,
-  "workspaces": [
-    "packages/*",
-    "examples/*"
-  ],
+  "workspaces": {
+    "packages": [
+      "packages/*",
+      "examples/*"
+    ],
+    "nohoist": [
+      "**/html-webpack-plugin",
+      "**/mini-css-extract-plugin"
```

Comment:
why? it seems like the only thing depending on this is webpack-config-anansi. are you worried hoisting will change the behavior of webpack-config-anansi (maybe it doesn't respect semver?). if so, why not nohoist all the packages from there?

Reply:
specifically this bugged out when there were changes needed with these plugins and it was resolving the storybook one

Reply:
it doesn't respect semver properly. this is a limitation (defective by design) of yarn workspaces and lerna

Reply:
basically semver is only looked at when installing, not resolving a module. the resolver assumes installation handled those checks correctly so it just walks directories.

in this case, what ends up happening, is these modules show up in the root level node_modules, but the one we want is in webpack-config-anansi/node_modules. If we avoid hoisting we can avoid this issue. This only occurs because of symlink issues with our monorepo and won't occur with external installs. which is why the fix is nohoist (i.e., monorepo specific fix for a monorepo specific problem)